### DISCUSSION TITLE
Pytester calling pytest_configure hook a second time

### DISCUSSION THREAD

**Discussion by Emantor** [pytest-dev/pytest#8099] [Q&A] [answered]
Hi,

within the labgrid pytest plugin configure hook we recently introduced registration of a logging handler. The registration asserts when trying to register the same handler a second time, so we run into errors when the pytester plugin is used, see https://github.com/labgrid-project/labgrid/issues/684. Is the `pytest_configure` hook expected to be callable multiple times? If so we should document this for all hooks.

Thanks,
Rouven

**Comment by RonnyPfannschmidt** [comment, marked as answer]
the hook is called exactly once per pytest configuration

when doing in process testing with pytester multiple pytest configurations are set up and used for test execution
